assets/src/scss/pages/post/_article-content.scss

```diff
@@ -100,8 +100,8 @@
   }
 
   p:not(.article-list-item-meta) a,
-  > ul li a,
-  > ol li a {
+  > article > ul li a,
```

Comment:
I wonder if we should remove the `ul` and `ol` from the selector.
In the bizarre scenario where editors add `<li>` elements using the HTML block instead of the List block, the styles are lost ([check here](https://www-dev.greenpeace.org/test-deimos/press/1109/duis-posuere-4/)).
I think, but not sure, that the selector could be even more simple and just be:  `.post-content li a`

Reply:
Unfortunately that wouldn't work because we have some blocks that use lists (such as the Articles block, or the new Posts/Actions List) and with this simpler selector then their links would also get the new styles and we don't want that

Reply:
I could add `> article > li a`, even though it's an edge case

Reply:
Works for me!

Reply:
Updated!